test/integration/test_seq_copy_int.py

```diff
@@ -51,7 +51,9 @@
      " --weight-tying-type src_trg_softmax"
      " --weight-init-scale=3.0 --weight-init-xavier-factor-type=avg"
      " --batch-size 2 --max-updates 2 --batch-type sentence --decode-and-evaluate 0"
-     " --checkpoint-interval 2 --optimizer adam --initial-learning-rate 0.01",
+     # Note: We se the checkpoint interval > max updates in order to make sure we create a checkpoint when reaching 
```

Comment:
@fhieber  Did you have something alone these lines in mind?

Reply:
yep, but typo: 'se' -> 'set'

Reply:
fixed